webpack.config.babel.js

```diff
@@ -13,14 +13,14 @@ const DEFAULT_SITE = PRODUCTION_MODE ? 'wsstelnet://ws.ptt.cc/bbs' : 'wstelnet:/
 export default {
   entry: {
     'pttchrome': [
-      './js/main.js',
-      './css/main.css',
-      './css/color.css'
+      './src/js/main.js',
+      './src/css/main.css',
+      './src/css/color.css'
     ]
   },
   output: {
-    path: path.join(__dirname, 'dist/'),
-    publicPath: '/dist/',
+    path: path.join(__dirname, 'dist/assets/'),
+    publicPath: '/assets/',
```

Comment:
Can you use relative path ('assets/') so the site can be put into non-root directory?
I was about to push a commit for this. Maybe you can do this along with this PR. Thanks!

Reply:
Good idea! I've done it in the last commit and rebased.
Testes development env and it's working fine. Could you test prod env on local?

Reply:
It looks like I always build prod and use ?site= to override the connecting site. It looks fine to me.

Reply:
Yeah b/c I didn't setup a nginx proxy on local (just use webpack-dev-server's proxy). Thanks for the help!